CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This file is used to list changes made in the last 3 major versions of the postg
 
 ## 11.1.1 - *2022-12-22*
 
+- Fix regex not matching usernames containing characters other than word characters
```

Comment:
Should this be under unreleased, not the prior version?

Reply:
Yup, @bmhughes please fix and then I'll approve!

Reply:
One day I'll update the system to use the PR body and not manual change log entries ... one day ... (I've been saying this to myself for 2+ years now)